tool/release/createnotes/Note.kt

```diff
@@ -75,7 +75,7 @@ class Note {
                 if (line.startsWith("##")) {
                     header += 1
                 } else if (header == 1) {
-                    goal.append(line)
+                    goal.append(line).append("\n")
```

Comment:
we were losing newlines, which ruined formatted PR goals

Reply:
@flyingsilverfin on second thought, although I agree with the importance of adding a newline for formatting purpose, it shouldn't be the responsibility of this method. As the name should imply, `getPRGoal` should be responsible for one thing: extracting the "goal" section ouf of a PR description and without any modification.

Which function should add the newline so that we can keep `getPRGoal` from violating its contract?